docs/no_installation_usage.rst

```diff
@@ -1,25 +1,76 @@
 ===================================
-Loading Premade Dataset Recipes
+Loading Datasets
 ===================================
 
-Loading a Premade Dataset Recipe
----------------------------------
+Loading a dataset
+-----------------
 
 With unitxt you can load a dataset without installing the package simply
-by using the following code (loading wnli dataset in 3 shot format):
+by using the following code:
 
 .. code-block:: python
 
   from datasets import load_dataset
 
-  dataset = load_dataset('unitxt/data', 'recipes.wnli_3_shot')
+  dataset = load_dataset('unitxt/data', 'card=cards.wnli')
 
-Loading Customized Data
---------------------------
+.. code-block:: python
+
+  print(dataset)
+  DatasetDict({
+    train: Dataset({
+        features: ['metrics', 'source', 'target', 'references', 'group', 'postprocessors'],
+        num_rows: 599
+    })
+    validation: Dataset({
+        features: ['metrics', 'source', 'target', 'references', 'group', 'postprocessors'],
+        num_rows: 36
+    })
+    test: Dataset({
+        features: ['metrics', 'source', 'target', 'references', 'group', 'postprocessors'],
+        num_rows: 71
+    })
+  })
+
+This prints the source text (input to the model) of the first sample in the training set:
+
+.. code-block:: python
+
+    print(dataset['train'][0]['source'])
+
+    'Given this sentence: I stuck a pin through a carrot. When I pulled the pin out, it had a hole., classify if this sentence: The carrot had a hole. is entailment, not entailment.\n'
```

Comment:
a general question: i wonder if the input text in the templates should be within two " characters. For the above example, it seems lacking. Though maybe LLMs don't care about that.

Reply:
We should improve the standard template for nil and then update this.